lib/kino_proxy/application.ex

```diff
@@ -3,9 +3,11 @@ defmodule KinoProxy.Application do
 
   @impl true
   def start(_type, _args) do
-    children = [KinoProxy.Client]
-    opts = [strategy: :one_for_one, name: KinoProxy.Supervisor]
+    children = [
+      {PartitionSupervisor,
+       child_spec: KinoProxy.Client.child_spec([]), name: KinoProxy.PartitionSupervisor}
```

Comment:
I think this works too!

```suggestion
       child_spec: KinoProxy.Client, name: KinoProxy.PartitionSupervisor}
```